CMakeLists.txt

```diff
@@ -1,6 +1,26 @@
 cmake_minimum_required(VERSION 3.18)
 project(tvm C CXX)
 
+if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  set(IS_WINDOWS 1)
+  message(STATUS "host is windows")
+  if (CMAKE_COMPILER_IS_GNUCC)
+    set(IS_MINGW 1)
+    if ("${CMAKE_BUILD_TYPE}" MATCHES "Debug")
+      add_definitions("-D_DEBUG")
+    endif ()
+  endif ()
+else()
+  set(IS_LINUX 1)
+  if ("${CMAKE_BUILD_TYPE}" MATCHES "Debug")
+    add_definitions("-D_DEBUG")
+  endif ()
+  add_definitions("-Dlinux")
+  message(STATUS "host is LINUX")
+endif ()
+
```

Comment:
Quick question: what are the following flags used for:
* `-D_DEBUG`
* `-Dlinux`